.drone.yml

```diff
@@ -15,6 +15,11 @@ steps:
     image: plugins/docker
     settings:
       repo: josaorg/community-web
+      build_args:
+        [
+          COMMUNITY_API_URL=https://community.api.prod.josa.ngo/v2,
+          TARGET_ENV=production,
+        ]
```

Comment:
The build_args are hardcoded with specific values which might not be suitable for different environments or scenarios. It is better to use environment variables for such configurations to enhance flexibility and security.

```diff - build_args: - [ - COMMUNITY_API_URL=https://community.api.prod.josa.ngo/v2, - TARGET_ENV=production, - ] + build_args_from_env: + [ + COMMUNITY_API_URL, + TARGET_ENV, + ] ```

Reply:
In our scenario, it is okay to use hard coded args.
Mainly because the auth nuxt package that we are using expects this value at build time. And there is no better way to provide this value at the moment.

Reply:
Sometimes its okay to use hard coded args, especially when they are build args. Build args are required during build time, meaning, you can inject them during runtime as an environment variable. Anyways, since then we have changed the structure of the .drone file and opted for using templates, this file is outdated.

We will need to update this branch as a whole.